### ISSUE TITLE
Проблема с button__hovered на touch

### ISSUE THREAD

**Issue by rteamx** [bem-site/bem-forum-content-ru#1315]
Суть в чем, используется common блоки на адаптивном проекте, у кнопок на тачах при первом клике вместо клика отрабатывает `__hovered`. Проблему можно повторить в том же `project-stab` - включаем в хроме эмуляцию какого-нибудь iphone 6 и жмем на кнопку `dropdown`.

Как можно вылечить? В [исходниках](https://github.com/bem/bem-components/blob/v5.0.0/common.blocks/button/button.js) кнопки модификатора `hovered` нет, откуда он приходит?

**Comment by tadatuta** [comment]
Предполагается, что на тачах должны собираться common+touch.

> В исходниках кнопки модификатора hovered нет, откуда он приходит?

`button` [наследует](https://github.com/bem/bem-components/blob/v5.0.0/common.blocks/button/button.js#L16) `control`, который и [предоставляет](https://github.com/bem/bem-components/blob/v5.0.0/desktop.blocks/control/control.js#L35-L41) `hovered`, правда это делается на уровне `desktop` и при использовании `common` появляться не должно.

**Comment by rteamx** [comment]
Точно, из `desktop` приходит. Но если отключить `desktop`, то будет та же проблема, но уже с `focused`.

**Comment by tadatuta** [comment]
То, что при клике выставляется модификатор `focused` — это ожидаемое поведение.
А вот то, что не происходит событие `click` — это баг. Завел [issue](https://github.com/bem/bem-components/issues/2033).

**Comment by belozer** [comment]
Как раз наткнулся на него тоже. Не работает с первого клика `_toggable_check`. 
`bem-components@3.1.3`
